ptclient/cts/spi/dx/cl/ctMultiChangeDxa.vue

```diff
@@ -23,27 +23,22 @@
                 >
               </el-form-item>
             </el-form>
-            <DxaHistoryItem :dx="dx"></DxaHistoryItem>
+            <dxa-history-item :dx="dx" />
           </el-card>
         </el-col>
       </el-carousel-item>
     </el-carousel>
   </el-row>
 </template>
 <script>
-import DxaHistoryItem from './ctDxaHistory'
+import DxaHistoryItem from './ctDxaHistory.vue'
 import ormDx from '@/models/Dx'
 export default {
   components: {
     DxaHistoryItem,
   },
-  data() {
-    return {}
-  },
```

Comment:
Unused `data` method. While a minor performance enhancement, Vue won't inspect and attempt to normalize component state if the `data` key is absent in a component.